Mage/src/main/java/mage/game/command/emblems/EmblemOfCard.java

```diff
@@ -0,0 +1,95 @@
+package mage.game.command.emblems;
+
+import mage.MageObject;
+import mage.abilities.AbilityImpl;
+import mage.cards.Card;
+import mage.cards.decks.DeckCardInfo;
+import mage.cards.mock.MockCard;
+import mage.cards.repository.CardCriteria;
+import mage.cards.repository.CardInfo;
+import mage.cards.repository.CardRepository;
+import mage.constants.Zone;
+import mage.game.command.Emblem;
+import mage.util.CardUtil;
+import org.apache.log4j.Logger;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class EmblemOfCard extends Emblem {
```

Comment:
Please add comment for an overview of what this class is for and how it is used

Reply:
While I wouldn't mind doing so (maybe while less tired so I can write it better), I went looking for examples of how such comments tend to be written, and not a single class I clicked on had any sort of explanation, just `@author` markings. Are you sure that's a thing we do here?

Reply:
Added explanation comment based on [my earlier explanation.](https://github.com/magefree/mage/pull/10498#issuecomment-1627449099)